src/app/maker/maker-unfilled-shift-condensed/maker-unfilled-shift-condensed.component.html

```diff
@@ -20,23 +20,14 @@
 
 
       <ion-item text-wrap color="light">
-        <ion-label text-wrap><b>Position(s):</b></ion-label>Filled / Total
+        <ion-label text-wrap><b>Position(s):</b></ion-label>
       </ion-item>
 
-        <!-- <ion-list color="light"> -->
-        <ion-card color="light">
-          <ul>
-            
-            <li>unfilled {{unfilled}}</li>
-            <!-- <li>{{shift.positions[1].position}}</li> -->
-          </ul>
+        <ion-card color="light"
+              *ngFor="let position of shift.positions let i=index" color="light" lines="none">
+                <ion-item slot="start" color="light" lines="none" expand="block">{{position.position}}<ion-item slot="end" color="light" lines="none">
+                  {{count}} out of 1</ion-item></ion-item>
```

Comment:
if "count" is, for instance, 5, do we see "5 out of 1"?
I think that maybe we could have a single count of filled positions at the end of the list of positions or something
How would we feel about a change of color for positions that aren't filled?